docs/modules/ROOT/pages/index.adoc

```diff
@@ -41,7 +41,7 @@ The link:https://docs.embassy.dev/embassy-net/[embassy-net] network stack implem
 The link:https://github.com/embassy-rs/nrf-softdevice[nrf-softdevice] crate provides Bluetooth Low Energy 4.x and 5.x support for nRF52 microcontrollers.
 
 === LoRa
-link:https://github.com/embassy-rs/lora-phy[lora-phy] and link:https://docs.embassy.dev/embassy-lora/[embassy-lora] supports LoRa networking on a wide range of LoRa radios, fully integrated with a Rust link:https://github.com/ivajloip/rust-lorawan[LoRaWAN] implementation.
+link:https://github.com/lora-rs/lora-rs[lora-rs] supports LoRa networking on a wide range of LoRa radios, fully integrated with a Rust link:https://github.com/ivajloip/rust-lorawan[LoRaWAN] implementation.
```

Comment:
rust-lorawan (protocol handling) was renamed and moved to lora-rs/lora-rs. And lora-phy (chip drivers) was merged some time later into lora-rs.

Reply:
Are you looking for additional explanation here added to the docs or is this just a note for reviewers?

Reply:
> Are you looking for additional explanation here added to the docs or is this just a note for reviewers?
It's a bit complicated:
1. You should drop at least github.com/ivajloip/rust-lorawan as it's now a redirect to https://github.com/lora-rs/lora-rs. This provides LoRaWAN protocol layer.
2. And lora-phy which provides radio interface, but it's also included within https://github.com/lora-rs/lora-rs
So basically, lora-rs is now a "batteries included" multi-crate repository.

Reply:
Ah! Okay, good points. How about:

link:https://github.com/lora-rs/lora-rs[lora-rs] supports LoRa networking on a wide range of LoRa radios, fully integrated with a Rust LoRaWAN implementation. It provides four crates: lora-phy, lora-modulation, lorawan-encoding, and lorawan-device.

I left out a detailed description of each crate to keep it terse, but this could optionally link to crates.io.

Here's what the lora-rs README provides:

> lora-modulation: LoRa modulation characteristics and a utility for calculating time on air
lora-phy: LoRa radio drivers which provide a PHY layer implementation
lorawan-encoding: encoding and decoding LoRaWAN packets
lorawan-device: a LoRaWAN device stack with non-blocking and async implementations

Reply:
A lot better though I would also append "... and basic examples for various development boards."

Reply:
I added this and a note about STM32WL and Semtech transceivers (same ones listed on the Embassy homepage).